app/services/data/get-individual-claim-details.js

```diff
@@ -87,19 +87,33 @@ module.exports = function (claimId) {
               return claimExpenses
             })
             .then(function (claimExpenses) {
-              return knex('Claim')
-                .join('ClaimChild', 'Claim.ClaimId', '=', 'ClaimChild.ClaimId')
-                .where({ 'Claim.ClaimId': claimId, 'ClaimChild.IsEnabled': true })
-                .select()
-                .orderBy('ClaimChild.Name')
-                .then(function (claimChild) {
-                  claimDetails = {
-                    claim: claim,
-                    claimExpenses: claimExpenses,
-                    claimChild: claimChild
-                  }
+              return knex('ClaimDeduction')
+                .where({'ClaimId': claimId, 'IsEnabled': true})
+                .then(function (claimDeductions) {
+                  var deductions = []
+                  var deductionTotal = 0
+                  claimDeductions.forEach(function (claimDeduction) {
+                    deductionTotal += claimDeduction.Amount
+                    deductions.push(claimDeduction)
+                  })
```

Comment:
move this and the ClaimExpense total to a separate service module to get Claim Total that can be more easily unit tested